### ISSUE TITLE
Store test_loss every 1000 iterations or so.

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#36]
Store this during training so we can analyze different learning rates, normalization etc.
